### ISSUE TITLE
cli needs to upgrade eslint-plugin-vue to match eslint 6

### ISSUE THREAD

**Issue by dkebler** [quasarframework/quasar#6253]
**Describe the bug**

The cli upgrade has upgraded to eslint 6.6 but did not upgrade eslint-plugin-vue which is still 5.3.2

So

1. now npm/yarn reports the wrong per dependency since 5.2.3 want eslint 5
2. being 5.2.3 it does not give access to the latest vue rules 
https://github.com/vuejs/eslint-plugin-vue/issues/899#issuecomment-569370526
e.g. https://eslint.vuejs.org/rules/valid-v-slot.html 

**To Reproduce**

do `quasar upgrade  -i`

```
warning " > eslint-loader@3.0.2" has unmet peer dependency "webpack@^4.0.0".
warning " > eslint-plugin-vue@5.2.3" has incorrect peer dependency "eslint@^5.0.0".
warning "eslint-plugin-vue > vue-eslint-parser@5.0.0" has incorrect peer dependency "eslint@^5.0.0".
```
**Expected behavior**

upgrade should upgrade eslint-vue-plugin to match eslint version so that there is no peer dependency warning

**Platform (please complete the following information):**
OS: linux
Node: 12
quasar: 1.7.4, now 1.8.3

I am attempting to  manually change the semiver of esling-plugin-vue to see if I can access the rules of 6+ but quasar cli should do this.

`"eslint-plugin-vue": "^6.1.2"`

**Comment by dkebler** [comment]
fyi my current dev dependencies showing in the package.json of my project is

```
  "devDependencies": {
    "@quasar/app": "^1.5.2",
    "@vue/eslint-config-standard": "^4.0.0",
    "babel-eslint": "^10.0.3",
    "eslint": "^6.6.0",
    "eslint-loader": "^3.0.2",
    "eslint-plugin-vue": "^6.1.2"
  },
```

where quasar set the eslint version and I set the plugin-vue version

**Comment by dkebler** [comment]
just confirming that I was able to lint fix entire project with 6.1.2 and my project is up running with upgraded plugin

**Comment by rstoenescu** [comment]
The Quasar upgrade command only upgrades Quasar related packages (ui, app, extras, app extensions). This does NOT includes eslint, obviously. We haven’t upgraded eslint to its v6 on the starter kit because of a babel-eslint issue. Since it’s been reported above that it now fully works, we’ll reassess the situation. But regardless, your own projects can be eslint upgraded or downgraded at will. Changing the starter kit won’t also upgrade the version of eslint that you are using (again, for obvious reasons — those are project specific, not q/app specific).
